Import services from ambanum/OpenTermsArchive (#98)
OpenTermsArchive/services-all: README.md

# Open Terms Archive - All services

> This repository is to be considered as a database of services declarations used by [Open Terms Archive](https://github.com/ambanum/OpenTermsArchive).

Reviewer: ```suggestion
The database of services tracked on [Open Terms Archive](https://opentermsarchive.org).
```

> You can learn more about [Open Terms Archive](https://github.com/ambanum/OpenTermsArchive) on the [dedicated repository](https://github.com/ambanum/OpenTermsArchive).

Reviewer: ```suggestion
These service declarations are used by the [Open Terms Archive engine](https://github.com/ambanum/OpenTermsArchive).
```
